contrib/cirrus/logformatter

```diff
@@ -310,6 +311,9 @@ END_HTML
         elsif ($line =~ /^Error:/ || $line =~ / level=(warning|error) /) {
             $line = "<span class='log-warn'>" . $line . "</span>";
         }
+        elsif ($line =~ /^panic:/) {
```

Comment:
Just double checking, I think it is indeed "panic", but had a moment of doubt and wondered if it's "Panic"?

Reply:
It actually is lower-case - look at the log file linked to in the commit message. There are already two upper-case `Panic`s handled by the formatter, but both are completely different circumstances: one is `Panic` followed by `It` or `BeforeEach`, the other is `Panic in [` (something with a square bracket). This is a completely new `panic` that I hadn't seen before.

Yes, I know it would be ideal to generalize the code, but so far each `Panic` is a different context that IMO deserves special handling.